chsdi/views/checker.py

```diff
@@ -1,16 +1,28 @@
 # -*- coding: utf-8 -*-
 
+from boto.exception import JSONResponseError
 from pyramid.view import view_config
 from pyramid.response import Response
+from pyramid.httpexceptions import HTTPInternalServerError
+from chsdi.views.files import FileView
 
 
-class Checker(object):
+class Checker(FileView):
 
     def __init__(self, request):
         self.request = request
+        self.dynamodb_table_name = 'geoadmin-file-'
+        self.bucket_key_name = 'geoadmin_file_storage_bucket66'
+        self.bucket_name = request.registry.settings['geoadmin_file_storage_bucket']
+        FileView.__init__(self, request)
 
     @view_config(route_name='checker')
```

Comment:
similarly to what we have done for `service-proxywms`, `/checker` should only check the sanity of mf-chsdi3, without the backends, and there should be a new route `/backend_checker` that checks the backends (and implicitely the sanity of `mf-chsdi3`.

Reply:
OK, done